cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -168,7 +168,16 @@ func (c *CStorVolumeReplicaController) syncHandler(
 	// Synchronize cstor volume total allocated and
 	// used capacity fields on CVR object.
 	// Any kind of sync activity should be done from here.
-	c.syncCvr(cvrGot)
+	err = c.syncCvr(cvrGot)
```

Comment:
better to call it as syncCVRStatus.. just syncCVR is very confusing

Reply:
I will update

Reply:
updated the function name